chain/trie/hasher.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"sync"
 
+	database "github.com/nknorg/nkn/chain/db"
```

Comment:
Why do we need the alias here?

Reply:
because there is a argument of function hash is db.